2021/46xxx/CVE-2021-46388.json

```diff
@@ -2,7 +2,7 @@
     "CVE_data_meta": {
```

Comment:
{} is not allowed for {'reference_data': [{'name': 'http://rejected', 'refsource': 'CONFIRM', 'url': 'http://rejected'}]}